server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/TcpClientInterface.js

```diff
@@ -236,5 +220,44 @@ class TcpClientInterface extends layerProtocol {
             }
         });
     }
+
+
+
+}
+
+
+/**
+ * @description This function returns the remote address configured .
+ * @param {String} tcpClientUuid : tcp-client uuid of the tcp-client instance
+ * @param {String} addressToBeDeleted : tcp-client address to be deleted.
+ * @param {String} remoteAddress : remote address of the tcp client .
+ * @returns {promise} list {paths}
+ **/
+function getPaths(tcpClientUuid, remoteAddress, addressToBeDeleted) {
+    return new Promise(async function (resolve, reject) {
+        let paths = [];
+        let remoteAddressPath;
+        let pathOfAddressToBeDeleted;
+        try {
+            if ("domain-name" in remoteAddress) {
```

Comment:
Please remove the hardcoded values. Instead , declare them as a constant in the module applicationPattern\onfModel\constants\OnfAttributes.js and utilize the declared constant. In this way , we can utilize the attribute name globally across many modules.

Reply:
Thank you for the comment.

Made changes as suggested , removed the hard coded values.